js/data.js

```diff
@@ -0,0 +1,189 @@
+ *     "checkin": "13:00",
+ *     "checkout": "14:00",
+ *     "features": [
+ *       "dishwasher",
+ *       "washer"
+ *     ],
+ *     "description": "Шале на воде, с фехтование, уникальные для собак.",
+ *     "photos": [
+ *       "https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg",
+ *       "https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg",
+ *       "https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg"
+ *     ]
+ *   },
+ *   "location": {
+ *     "lat": "35.65264",
+ *     "lng": "139.78088"
+ *   }
+ *}
+ *
+ * @param {number} userNumber - идентификатор пользователя
+ * @return {object} - объект объявления со случайными данными.
+ */
+const createAdvertisement = (userNumber) => {
+  const location = getRandomLocation(LATITUDE_RANGE, LONGITUDE_RANGE);
+  const type = getRandomValueFromArray(BUILDING_TYPES);
+
+  return {
+    author: {
+      avatar: `img/avatars/user${getAvatarNumber(userNumber, LIMIT_NUMBER_FOR_ZERO)}.png`,
+    },
+    offer: {
+      title: TITLES_VALUE[type],
+      address: `${location.lat}, ${location.lng}`,
+      price: generateRandomNumber(PRICE_RANGE.MIN, PRICE_RANGE.MAX),
+      type,
+      rooms: generateRandomNumber(ROOMS_RANGE.MIN, ROOMS_RANGE.MAX),
+      guests: generateRandomNumber(GUESTS_COUNT.MIN, GUESTS_COUNT.MAX),
+      checkin: getRandomValueFromArray(CHECK_VALUES),
+      checkout: getRandomValueFromArray(CHECK_VALUES),
+      features: getSomeValuesFromArray(FUTURES),
+      description: DESCRIPTIONS_VALUE[type],
+      photos: getSomeValuesFromArray(PHOTOS_PATH),
+    },
+    location,
+  };
+};
+
+/**
+ * Генерирует массив объявлений
+ *
+ * Пример:
+ * generateAdvertisements(1);
+ * result = [{
+ *   "author": {
+ *     "avatar": "img/avatars/user01.png"
+ *   },
+ *   "offer": {
+ *     "title": "Бунгало от 1 до 6 людей",
+ *     "address": "35.65264, 139.78088",
+ *     "price": "141802",
+ *     "type": "bungalow",
+ *     "rooms": "15",
+ *     "guests": "6",
+ *     "checkin": "13:00",
+ *     "checkout": "14:00",
+ *     "features": [
+ *       "dishwasher",
+ *       "washer"
+ *     ],
+ *     "description": "Шале на воде, с фехтование, уникальные для собак.",
+ *     "photos": [
+ *       "https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg",
+ *       "https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg",
+ *       "https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg"
+ *     ]
+ *   },
+ *   "location": {
+ *     "lat": "35.65264",
+ *     "lng": "139.78088"
+ *   }
+ *}]
+ *
+ * @param {number} advertisementCount - количество объявлений
+ * @return {array} - массив объявлений со случайными данными.
+ */
+const generateAdvertisements = (advertisementCount) => {
```

Comment:
```suggestion
const generateAdvertisements = (count) => {
```

из названия функции и так понятно что это, можно просто count